db/migrations/versions/13d601bfaee1_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
+
+Revision ID: 13d601bfaee1
+Revises:
+Create Date: 2022-07-20 12:59:17.168742
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils  # noqa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "13d601bfaee1"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "account",
+        sa.Column(
+            "id",
+            sqlalchemy_utils.types.uuid.UUIDType(binary=False),
+            nullable=False,
+        ),
+        sa.Column("email", sa.String(), nullable=False),
```

Comment:
should we add a character limit here? 🤔

Reply:
how much would be suitable?

Reply:
255 i've seen generally being used? seems small and big enough?